stdlib/2/sets.pyi

```diff
@@ -0,0 +1,60 @@
+# Stubs for sets (Python 2)
+from typing import Any, Callable, Hashable, Iterable, Iterator, Mapping, Optional, TypeVar, Union
+
+_T = TypeVar('_T')
+Setlike = Union['BaseSet', Iterable]
+
+class BaseSet(Iterable[_T]):
+    def __init__(self) -> None: ...
+    def __len__(self) -> int: ...
+    def __repr__(self) -> str: ...
+    __str__ = ...  # type: Callable[[BaseSet[_T]], str]
```

Comment:
I think this should just be expressed as `def __str__(self) -> str: ...`. It looks like your stub generator made it a variable because in the implementation it's an alias for `__repr__`. The same thing goes for `__copy__`, `__le__`, and `__ge__` below.

In any case this specific method is redundant because object already has the same method.